examples/node-node-version/.node-version

```diff
@@ -0,0 +1 @@
+v14.18.1
```

Comment:
I think the example should include a newer non-deprecated version of node. Version 18+